bin/utils.py

```diff
@@ -56,10 +67,46 @@ def load_config(self, filename):
             job_set = job_set.split(',')
             for job_item in job_set:
                 self.jobs[tenant_item].append(job_item)
-        
+
         # Grab connector sync path
-        self.sync_exec = ar_config.get('connectors','sync_exec')
-        self.sync_path = ar_config.get('connectors','sync_path')
+        self.sync_exec = ar_config.get('connectors', 'sync_exec')
+        self.sync_path = ar_config.get('connectors', 'sync_path')
+
+        # Grab run mode
+        self.mode = ar_config.get('default', 'mode')
+
+        # Grab mapping info (will be removed in the near future)
+        self.n_alt = ar_config.get('connectors', 'n_alt')
```

Comment:
tests break here. I think you should remove lines 79-85 (`connectors` and `sampling` sections) or we should add them in `utils_test.py`.